sdk/python/kfp/v2/compiler/compiler_test.py

```diff
@@ -379,6 +379,11 @@ def my_pipeline(datasets):
           pipeline_root='dummy',
           output_path='output.json')
 
+  def test_import_kfp_v2_component_raise_warning(self):
+    with self.assertWarnsRegex(DeprecationWarning,
+                               'Module `kfp.v2.compoennts` is deprecated'):
```

Comment:
Will we never introduce v2 components again?
It sounds a little weird we deprecate the v2 version of components and ask people to use v1
: )

Reply:
So maybe something like 'Module kfp.v2.components has been merged back to kfp.components. Please directly import kfp.components' makes better sense?

Reply:
The current plan is that only compiler will be under v2 namespace. 
But I think I see your point, maybe we can keep such a v2 alias forever? 

@numerology, @neuromage, WDYT? 

/hold

Reply:
> So maybe something like 'Module kfp.v2.components has been merged back to kfp.components. Please directly import kfp.components' makes better sense?

I mean even the code is merged back to v1, we can still keep a v2 alias so that users can import either way without seeing any difference?

Reply:
Yes but my understanding is v2 namespace will be eventually merged back to v1 in its entirety, right?

Reply:
> Yes but my understanding is v2 namespace will be eventually merged back to v1 in its entirety, right?

Ah, right. So how about we allow the v2.components import without throwing any warning for now. But add the warning later when we're ready to remove the entire v2 namespace? Like do it once instead of one module at a time.

Reply:
That SGTM as well.

Reply:
Removed the warning part.

Reply:
/unhold